packages/cli/src/utils/is-blitz-root.ts

```diff
@@ -0,0 +1,70 @@
+import {readJSON} from 'fs-extra'
+import pkgDir from 'pkg-dir'
+import {resolve} from 'path'
+
+export enum IsBlitzRootError {
+  notBlitz,
```

Comment:
can we use `PascalCase` for enum values? we've used that convention in a few other places in the CLI